default/scripting/species/common/planet_shields.macros

```diff
@@ -37,7 +37,8 @@ PROTECTION_FOCUS_SHIELDS
             ]
             stackinggroup = "FOCUS_PROTECTION_SHIELDS_STACK"
             priority = [[TARGET_AFTER_2ND_SCALING_PRIORITY]]
-            effects = SetMaxShield value = Value * 2 accountinglabel = "FOCUS_PROTECTION_LABEL"
+            effects = SetMaxShield value = Value * 2
+            accountinglabel = "FOCUS_PROTECTION_LABEL"
```

Comment:
This is misleading, I think. the accounting label is part of the effect, so should be on the same line or indented to indicate this. With this intenting, it looks like it's part of the effectsgroup.

Is there a reason to put it in the effect, and not in the effectsgroup like https://github.com/freeorion/freeorion/pull/3643/files#diff-9bb016b04ee777c345a07faf65c3cf8794ce7d2e69a2c90e14b06b86dea146bdR37

Reply:
oh that is still somewhat unclear to me, sorry. it just seemed that in most cases the accounting label went on its own line of code with nothing else. this looks like 'set the shield value to twice the value of the accounting label' and is a little confusing

Reply:
it goes on its own line if it's one of the parameters to the EffectsGroup, but if it's a parameter to the effect, then it would probably go on the same line, or at least indented to make that clearer

Reply:
Ahh heureka ..! yeah that explains some things